src/Pages/Shared/ConfigMapSecret/ConfigMapSecretContainer.tsx

```diff
@@ -230,6 +229,7 @@ export const ConfigMapSecretContainer = ({
             })
 
             if (data.draftData) {
+                setAreCommentsPresent(data.draftData.commentsCount > 0)
```

Comment:
We should not call setState in useMemo

Reply:
fixed in https://github.com/devtron-labs/dashboard/pull/2130